master/internal/trial.go

```diff
@@ -225,6 +226,13 @@ func newTrial(
 	}
 }
 
+func (t *trial) doKillTrial(ctx *actor.Context) {
```

Comment:
Why not just 
```suggestion
func (t *trial) killTrial(ctx *actor.Context) {
```

Reply:
we have a message defined with that name in the same scope. I think they were colliding. i"ll check

Reply:
yeah forgot about the message type.